benefits/pensions.md

```diff
@@ -0,0 +1,38 @@
+# Pensions
+
+We provide a pension plan which everyone is automatically enrolled on.
+
+You will be enrolled as soon as you pass your probation period which will be stated your employment contract.
+
+## Contributions
+
+As your employer we will contribute towards your pension alongside your own contribution. The Government is currently rolling out a workplace pension scheme which we are following. This will see both employer and employee contributions scale up over the next 3 years, subject to parliamentary approval and guidelines.
+
+<table>
+  <tr>
+    <th>&nbsp;</th>
+    <th>Our (employer) contribution</th>
+    <th>Your (employee) contribution</th>
+  </tr>
+  <tr>
+    <th>Current</th>
+    <td>1%</td>
+    <td>1%</td>
```

Comment:
Right gotcha, yes these are the default amounts. As noted further down you can choose to increase if you want. Do you have a suggestion for making it clearer here?